test/custom-options.test.js

```diff
@@ -1,4 +1,4 @@
-const gotScraping = require('../src');
+const gotScraping = require('../dist');
```

Comment:
I think we prefer testing source directly, but this can be done later.

Reply:
WDYM?

Reply:
https://github.com/apify/apify-storage-local-js/blob/f8bae3875e2b6e954ea843764d6a07dd46bba566/test/index.test.ts#L4

Reply:
> [apify/apify-storage-local-js@`f8bae38`/test/index.test.ts#L4](https://github.com/apify/apify-storage-local-js/blob/f8bae3875e2b6e954ea843764d6a07dd46bba566/test/index.test.ts#L4) `import` from `src`, not `dist`. Which would require rewrite of tests to TS, which is why I said that we can do it later.

It would only require changing the jest config to a ts one, like I did in browser-pool/storage-local. Then you can import from src directly (and skip needing to build before testing)